projects/plugins/jetpack/_inc/client/state/site/reducer.js

```diff
@@ -412,3 +423,26 @@ export function getConnectedPlugins( state ) {
 	const plugins = get( state.jetpack.siteData, [ 'data', 'site', 'connectedPlugins' ], [] );
 	return plugins.filter( plugin => 'jetpack' !== plugin.slug );
 }
+
+/**
+ * Returns true if the site has a subscription to a Backup product or to a plan that includes Backup.
+ *
+ * @param   {object} state - Global state tree
+ * @returns {boolean} True if the site does have Backup
+ */
+export function siteHasBackup( state ) {
```

Comment:
I found the name a bit confusing; this is not about whether the site has a backup or not, but about the site's plan. Maybe we could rename this to `siteHasBackupPlan`?

Reply:
Fair point.